src/Fable.Cli/ProjectCracker.fs

```diff
@@ -488,6 +488,7 @@ let copyFableLibraryAndPackageSources (opts: CrackerOptions) (pkgs: FablePackage
 
             let defaultFableLibraryPaths =
                 [ "../../../fable-library/"               // running from nuget tools package
+                  "../../../../fable-library/"            // running from Visual Studio
                   "../../../../../build/fable-library/" ] // running from bin/Release/netcoreapp3.1
```

Comment:
I'm ok with this but CI is failing and it may be related. I guess it's detecting the `src/fable-library` directory without the compiled files? Can you try adding the new path after `../../../../../build/fable-library/` to see if this fixes the CI build? BTW, it's been a while since I haven't used Visual Studio, but sometimes I debug with VS Code or Rider without issue. Does not VS put the executable fable.dll in `src/Fable.Cli/bin/Debug/netcoreapp3.1`?

Reply:
> I guess it's detecting the `src/fable-library` directory without the compiled files?

Yes, that's actually what I intended because I misunderstood what it was supposed to be... My bad.

> Does not VS put the executable fable.dll in `src/Fable.Cli/bin/Debug/netcoreapp3.1`?

It does.

I did not realize using `dotnet fsi build.fsx` before I could use the IDE to debug was mandatory.
(BTW, is there not a target on build.fsx just to build without testing or anything else? I can't seem to find it)